### DISCUSSION TITLE
[Shaman] Temporary workaround for spirit wolfs dmg/healing

### DISCUSSION THREAD

**Discussion by eragonx78** [chromiecraft/chromiecraft#6037] [General]
Hello,
Shaman spirits wolfs are causing a lot of frustration and issues for arena and a lot of players are complaining on discord about it.
As it was recommended to me i open this discussion so we find a temporary fix until the final solution is done.
Here is the related issue on AzerothCore with more details on the subject: 
https://github.com/azerothcore/azerothcore-wotlk/issues/16997

**Comment by shaszun** [comment]
I know everything is balanced for lvl 80, but Rogues and Warriors are also having OP spells or talents which more or less make it useless to get resilience at all or even play arenas. Me as a plater, with around 7k armor and  250 resilience getting 6k dmg from a rogue in 3 seconds is just stupid. So far I haven't received a single stun from a rogue in PvP, best case sap or blind as a CC. Prot warriors are the same, big dmg in short time.

**Comment by eragonx78** [comment]
What does this have to do with the subject ? Please dont disturb the original discussion with your things that are not useful to the original subject.

**Comment by Never6** [comment]
Their is currently a lot of issues plaguing pvp, but in general I think damage overall is just wrong compared to classic 
I watched plenty of prepatch duels and prepatch classic was with sunwell gear and 3.3.5 talents so basically what we are playing currently except they have better gear and damage wasn't nearly as high as it is currently and sunwell gear is way better than s1
